include/notifier.php

```diff
@@ -553,18 +538,6 @@ function notifier_run(&$argv, &$argc){
 		if (dbm::is_result($r)) {
 			logger('pubdeliver '.$target_item["guid"].': '.print_r($r,true), LOGGER_DEBUG);
 
-			// throw everything into the queue in case we get killed
```

Comment:
This is the obsolete "deliverq" code that we don't need anymore with the worker.